components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2159,6 +2159,9 @@ type Identity struct {
 	PrimaryEmail string   `json:"primaryEmail,omitempty"`
 	Readonly     bool     `json:"readonly,omitempty"`
 	Tokens       []*Token `json:"tokens,omitempty"`
+
+	// The last time this entry was touched during a signin.
+	LastSigninTime string `json:"last_signin_time,omitempty"`
```

Comment:
Can we make this a `time.Time` on the protocol level? The RPC should then convert appropriately when it either sends the request, or receives it.